dev-docs/bidders/preciso.md

```diff
@@ -13,51 +15,115 @@ prebid_member: true
 floors_supported: true
 safeframes_ok: true
 schain_supported: true
-userIds: id5Id, identityLink, pubProvidedId
-pbs_app_supported: true
+userIds: sharedId
+deals_supported: false
+pbs_app_supported: false
 coppa_supported: true
-multiformat_supported: will-bid-on-any
-ortb_blocking_supported: partial
```

Comment:
If you don't support it than state it with false `ortb_blocking_supported: false`

Reply:
Have you added the line back in?

Reply:
yes.